src/poetry/core/packages/project_package.py

```diff
@@ -49,11 +49,11 @@ def to_dependency(self) -> Union["DependencyTypes"]:
         return dependency
 
     @property
-    def python_versions(self) -> Union[str, "VersionConstraint"]:
+    def python_versions(self) -> str:
         return self._python_versions
 
     @python_versions.setter
-    def python_versions(self, value: Union[str, "VersionConstraint"]) -> None:
+    def python_versions(self, value: str) -> None:
```

Comment:
If we are sure about the type, we can remove `or value == VersionRange()` some lines below as well as the local import of `VersionRange`. At least, I could not find any part in the code that passes a `VersionConstraint`.

Reply:
Fair point. Fixed.